docs/core/DelegationManager.md

```diff
@@ -126,7 +126,7 @@ Allows the caller (an AVS) to register an `operator` with itself, given the prov
 * Pause status MUST NOT be set: `PAUSED_OPERATOR_REGISTER_DEREGISTER_TO_AVS`
 * `operator` MUST already be a registered Operator
 * `operator` MUST NOT already be registered with the AVS
-* `operatorSignature` must be a valid, unused, unexpired signature from the `operator`
+* `operatorSignature` must be a valid, unused, unexpired signature from the `operator`. The signature is an ECDSA signature by the operator over the [`OPERATOR_AVS_REGISTRATION_TYPEHASH`](../../src/contracts/core/DelegationManagerStorage.sol#L29). Expiry is a utc timestamp in seconds. Salt is used only once per signature to prevent replay attacks.
```

Comment:
Good change, but I'd request that you remove the specific line number link, since that can get outdated quickly. I've just been referring to the constants without a link - it's CTRL-F-able at least